### PR TITLE
Adding typings to package copy

### PR DESCRIPTION
**Pull request by ulziibay**
if a selected packages has  `@typings` package in DefinitelyTyped as determined by https://github.com/returntocorp/echelon-backend/pull/2205, then display it to copy field
**when long package names are including @types**
screen shot 2018-12-06 at 9 07 26 pm
**when including @types**
screen shot 2018-12-06 at 9 07 30 pm
**when no @types found:**
screen shot 2018-12-06 at 9 07 47 pm

### CODE REVIEW

**Comment by ulziibay** [comment]
Addresses https://github.com/returntocorp/preflight-extension/issues/103, but not in checklist

**Comment by lediur** [comment]
Comments from Slack:

- Can we try making it a toggle button like `Use npm` and see how that looks?
- Would rather not show the option at all if we don't have `@types` information to show since there's limited space to begin with.

File: src/content/headsup/PackageCopyBox.tsx [hunk 1]
```diff
@@ -53,14 +58,27 @@ export class PackageCopyBox extends React.PureComponent<PackageCopyBoxProps> {
       packages,
       selectedPackage,
       packageManager,
-      onChangePackageManager
+      typesInclusion,
+      onChangePackageManager,
+      onChangeTypesInclusion
     } = this.props;
 
+    const typesFound: boolean = selectedPackage.types
```
Comment (lediur) [inline src/content/headsup/PackageCopyBox.tsx]:
```suggestion
    const typesFound: boolean = selectedPackage.types != null && selectedPackage.types.package_name != null
```

Comment (lediur) [reply src/content/headsup/PackageCopyBox.tsx]: would have to delete following line too

File: src/content/headsup/PackageCopyBox.tsx [hunk 2]
```diff
@@ -194,16 +217,20 @@ export class PackageCopyBox extends React.PureComponent<PackageCopyBoxProps> {
     event?: React.SyntheticEvent<HTMLElement>
   ) => this.props.onSelectPackage(item, event);
 
-  private handleCopy = (registry: string, packageName: string) => (
-    e: React.MouseEvent<HTMLElement>
-  ) => {
+  private handleCopy = (
+    registry: string,
+    packageName: string,
+    typesInclusion: boolean
```
Comment (lediur) [inline src/content/headsup/PackageCopyBox.tsx]: grammar nit: can we name this `includeTypesCommand` or something? should be active voice

Comment (lediur) [reply src/content/headsup/PackageCopyBox.tsx]: applies to below as well. `typesInclusion` is less clear to me

**Comment by ulziibay** [comment]
added logging as well

**Comment by ulziibay** [comment]
@lediur: making the button non-minimal at least fixes the z-index. LMK. Addressed your other requests as well. 
screen shot 2018-12-11 at 2 50 23 pm

**Comment by ulziibay** [comment]
@lediur addressed the crowdedness per you suggestion:
screen shot 2018-12-12 at 11 25 33 am
screen shot 2018-12-12 at 11 25 36 am

**Comment by ulziibay** [comment]
cc: @dlukeomalley

**Comment by ulziibay** [comment]
screen shot 2018-12-12 at 2 03 08 pm
screen shot 2018-12-12 at 2 03 10 pm
screen shot 2018-12-12 at 2 03 14 pm

**Comment by ulziibay** [comment]
screen shot 2018-12-12 at 3 43 28 pm

**Pull request merged**